crates/swift-bridge-build/src/generate_core.rs

```diff
@@ -143,6 +143,8 @@ fn conform_to_vectorizable(swift_ty: &str, rust_ty: &str) -> String {
     format!(
         r#"
 extension {swift_ty}: Vectorizable {{
+    public typealias Elem = {swift_ty}
+
```

Comment:
Add the new `typealias` up top, and below the method implementation - it's the same procedure for all of these generated code segments.